chatlearn/models/vllm_module.py

```diff
@@ -252,6 +257,7 @@ def init(self):
         initialize_vllm(extra_args_provider=self.add_extra_args,
                         ignore_unknown_args=True,
                         args_dict=self.model_args)
+        self.parallel_config.rank = torch.distributed.get_rank()
```

Comment:
where used?

Reply:
When profiling gpu blocks, model_runner in vLLM worker uses to get pp layer partition. 

ref: 

1. https://github.com/vllm-project/vllm/blob/v0.6.3.post1/vllm/worker/model_runner.py#L1276
2. https://github.com/vllm-project/vllm/blob/v0.6.3.post1/vllm/config.py#L549